exerciseOne.js

```diff
@@ -0,0 +1,5 @@
+var date = new Date();
+var n = date.toDateString();
+var time = date.toLocaleTimeString();
```

Comment:
another way you could write this is:
```
var date = new Date(),
      n = date.toDateString(),
      time = date.toLocaleTimeString();

```
In this way you avoid repeating `var var var` (note the commas instead of the semicol on the first 2 declarations